mlibrary/my_account
# Modal

# Overview
A modal needed to be created to be used as an `Are you sure?` pop up for Settings when confirming to delete your checkout history ([see mockup](https://projects.invisionapp.com/d/main/default/#/console/20707215/451712643/preview)).

Resolves #153.

## Testing
- Run the tests to make sure they pass: `docker-compose run web bundle exec rspec`
  - Break the tests to double-check that they work
- Go to [Settings](http://localhost:4567/settings) and try to delete checkout history to see the modal pop up.
- Click the `X` or `Cancel` buttons to see if the modal closes.
- Click the `Delete history` to see if it continues the `POST`.
